src/reflect/scala/reflect/internal/SymbolTable.scala

```diff
@@ -94,12 +94,6 @@ abstract class SymbolTable extends macros.Universe
 
   def settings: MutableSettings
 
-  @deprecated("Interactive is implemented with a custom Global; this flag is ignored", "2.11.0") def forInteractive = false
-  @deprecated("Scaladoc is implemented with a custom Global; this flag is ignored", "2.11.0")    def forScaladoc = false
```

Comment:
So, er, how I am supposed to test whether the current compiler is used as a Scaladoc run in a cross-compilable way from 2.10 to 2.13? I used to rely on this deprecated `forScaladoc` test. If 2.10 is not a concern, I can do `global.isInstanceOf[doc.ScaladocGlobal]`, which is what we do in Scala.js 1.x. But in 0.6.x it seems I need to use dark magic:
```scala
  @inline final def forScaladoc: Boolean =
    global.forScaladoc

  implicit final class GlobalCompat(
      self: Compat210Component.this.global.type) {

    def forScaladoc: Boolean =
      self.isInstanceOf[ScaladocGlobalCompat.ScaladocGlobalCompat]
  }

  object ScaladocGlobalCompat {
    object Compat {
      trait ScaladocGlobal
    }

    import Compat._

    object Inner {
      type ScaladocGlobalCompat = ScaladocGlobal
    }

    type ScaladocGlobalCompat = Inner.ScaladocGlobalCompat
  }
```

Reply:
See https://github.com/scala-js/scala-js/pull/3564.

Reply:
I assumed `instanceof`, but I'll spend some of U.S. Presidents Day to understand the 0.6 issue. There's no agenda behind removing the members except decruftification. Since it's late in the cycle, I'll PR to restore them in case that's the right thing.

Reply:
in general, we're not trying to support cross-compiling on 2.10 anymore. especially not when compiler internals are involved

if anything, we're actively campaigning for everyone to drop 2.10 support, as it's become a drag on the entire ecosystem

it's not that we want to put Scala.js in a tight spot, but, I'm surprised to learn (well, to be reminded, I suppose) that you're even still targeting 2.10 for new Scala.js releases, even point releases. I don't think it should be driving any decisions in here in scala/scala, even little ones

Reply:
(over in Fortify-for-Scala, I just introduce a tiny a bit of version-specific source any time I run into some little hard-to-cross-compile like this. admittedly, I'm only going back as far as 2.11. still, cross-compiling without any version-specific source at all seems like something of a parlor trick rather than an engineering requirement)

Reply:
It's OK. I found a trick and it's merged. I don't even know why I posted here, actually. I think I began my comment at a time where I thought I wouldn't find a solution for this one, but I guess I underestimated my twisted mind :stuck_out_tongue_winking_eye: 

> it's not that we want to put Scala.js in a tight spot, but, I'm surprised to learn (well, to be reminded, I suppose) that you're even still targeting 2.10 for new Scala.js releases, even point releases.

We're at the root of the dependency graph. We can only drop 2.10 when everyone else has done it before us. Otherwise people complain that supporting Scala.js for their libraries is too hard.

We did drop 2.10 in Scala.js 1.x, though.

Reply:
I prefer to have a paper trail of feedback. Also, breaking stuff makes me feel like what I do really matters.

Reply:
> We're at the root of the dependency graph. We can only drop 2.10 when everyone else has done it before us

I'll continue my campaign of actively discouraging everybody to drop 2.10, then 👍